### PR TITLE
Add layout file with navbar

### LINKED ISSUE #27: Navigation Bar (navbar)
With the following tabs:
Home
Subscribe

### PR DESCRIPTION
**Pull request by Yarinh55**
fix #27

### CODE REVIEW

**Review by lmilbaum — CHANGES_REQUESTED** [review]
I can't see the navbar.  You should render html file when browsing to home page

**Comment by lmilbaum** [comment]
How did you test this PR?

**Comment by Yarinh55** [comment]
@lioramilbaum 
sorry, my mistake. 
I miss it at push to GH

File: Vagrantfile [hunk 1]
```diff
@@ -3,4 +3,4 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell",
   path:"scripts/bootstrap.sh"
   config.vm.network "forwarded_port", guest: 5000, host: 5000 
-end
+end
```
Comment (lmilbaum) [inline Vagrantfile:6]: how this change relates to this PR?

File: app.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 app = Flask(__name__)
 
-
 @app.route('/')
-def hello_world():
-    return 'Hello World'
+def Home():
+    return render_template('Home.html')
+	
+@app.route('/')
```
Comment (lmilbaum) [inline app.py:10]: This change is also unrelated. Remove it.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

**Pull request closed without merging**